generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management.component.ts.ejs

```diff
@@ -112,8 +112,4 @@ export class <%= entityAngularName %>Component implements OnInit, OnDestroy {
         }
     }
     <%_ }} _%>
-
-    protected onError(errorMessage: string) {
-        this.jhiAlertService.error(errorMessage, null, null);
```

Comment:
`jhiAlertService` variable can now be removed from `infinite-scroll-template.ejs`, `no-pagination-template.ejs` and `pagination-template.ejs`.

And `JhiAlertService` import can be removed after that from `entity-management.component.ts.ejs`.